lib/components/ToasterDialog/ToasterDialog.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import type { Meta, StoryObj } from 'storybook-solidjs'
+import { ToastContainer } from 'react-toastify'
+
+import { default as ToasterDialogComponent } from './ToasterDialog'
+import { DialogProvider } from '../../context'
+import React from 'react'
+import Utils from '../../utils'
+import Button from '../Button'
+import { CloseButton } from '../index'
+
+const meta: Meta<typeof ToasterDialogComponent> = {
+  component: ToasterDialogComponent,
+  title: 'Components/ToasterDialog'
+}
+export default meta
+
+type Story = StoryObj<typeof ToasterDialogComponent>
+
+const DialogStory = (args: object) => (
+  <DialogProvider>
+    <div style={{ display: 'flex', gap: '20px' }}>
+      <Button
+        onClick={() =>
+          Utils.toastSuccess(
+            'This is very long Success message that should be displayed in the toaster dialog to test the overflow of the text. '
+          )
+        }
+      >
+        Dialog Success
+      </Button>
+      <Button
+        onClick={() =>
+          Utils.toastError(
+            'This is very long Error message that should be displayed in the toaster dialog to test the overflow of the text. '
+          )
+        }
+      >
+        Dialog Error
+      </Button>
+    </div>
+
+    <ToasterDialogComponent {...args} />
+  </DialogProvider>
+)
+
+const ToasterStory = () => (
```

Comment:
let's fix the styling to be the same as in the projects:

in the story:
<img width="1028" alt="Screenshot 2024-07-24 at 11 11 34" src="https://github.com/user-attachments/assets/6b8e4bbd-544d-40d8-ae41-b8fb0a660e16">

in projects:
<img width="447" alt="Screenshot 2024-07-24 at 11 09 55" src="https://github.com/user-attachments/assets/731f52b2-8e75-4765-8482-518b16071b5f">